src/extractor/guidance/sliproad_handler.cpp

```diff
@@ -78,10 +78,14 @@ operator()(const NodeID, const EdgeID source_edge_id, Intersection intersection)
             // a one-sided sliproad, however, the non-sliproad can be considered `obvious`. Here we
             // assume that this could be the case and check for a potential sliproad/non-sliproad
             // situation.
-            const auto intersection_following_index_one =
-                findNextIntersectionForRoad(intersection_node_id, intersection[1], NULL);
-            const auto intersection_following_index_two =
-                findNextIntersectionForRoad(intersection_node_id, intersection[2], NULL);
+            NodeID intersection_node_one, intersection_node_two;
```

Comment:
It seems like `findNextIntersectionForRoad` takes a `NodeID *` writing into it. I think with this change a `NodeID & out` would be fine, too. And then you can change the logic in the lambda.